core/templates/dev/head/css/oppia.css

```diff
@@ -110,6 +110,11 @@ body {
   width: 100%;
 }
 
+/*Added to not show warning content when Javascript is not enabled. */
```

Comment:
In general, we eschew the use of !important in CSS, since it basically messes around with the usual CSS priority order. However, I can see why we might need to make an exception here, and I'm fine with this.

Having said that, could you please add a link to whatever reference you used to come up with this solution, and a bit more explanation of why it's needed? (It wasn't obvious to me at the outset how to connect the dots between "JS is not enabled" and "warning content is not shown".) That will give useful context to future developers.

Finally, a nit: please add a single space after the opening /*.

Thanks!